tests/integrationtest/r/planner/core/rule_constant_propagation.result

```diff
@@ -270,9 +270,9 @@ WHERE t3.k = t2.k AND t3.b = t2.b
 )
 AND t2.k = 1;
 id	estRows	task	access object	operator info
-HashJoin_17	7.99	root		semi join, left side:TableReader_21, equal:[eq(planner__core__rule_constant_propagation.t2.b, planner__core__rule_constant_propagation.t3.b)]
+HashJoin_17	7.99	root		semi join, left side:TableReader_21, equal:[eq(planner__core__rule_constant_propagation.t2.k, planner__core__rule_constant_propagation.t3.k) eq(planner__core__rule_constant_propagation.t2.b, planner__core__rule_constant_propagation.t3.b)]
```

Comment:
Too much equal condtions will be fixed in https://github.com/pingcap/tidb/issues/62991